Makefile

```diff
@@ -73,6 +77,22 @@ kube-clean:
 	kubectl delete rc deis-${SHORT_NAME}
 	kubectl delete svc deis-${SHORT_NAME}
 
+kube-delete:
```

Comment:
yeah I was going to delete them. I didnt know if you guys have specific scripts using those or not. I just know we have kube-create/delete in workflow and other repos and its pretty handy. I'll update the PR in just a sec